pkg/security/probe/process_resolver.go

```diff
@@ -642,8 +645,13 @@ func (p *ProcessResolver) SetProcessPath(fileEvent *model.FileEvent, ctx *model.
 		return "", &ErrInvalidKeyPath{Inode: fileEvent.Inode, MountID: fileEvent.MountID}
 	}
 
-	fileEvent.SetPathnameStr(pathnameStr)
-	fileEvent.SetBasenameStr(path.Base(fileEvent.PathnameStr))
+	// fileless ?
+	if isFileExecution(fileEvent) {
```

Comment:
fileEvent.Inode == 0 is already tested line 630 maybe we could just test the mountID here instead of using the function

Reply:
I want to keep using the func instead of just `fileEvent.MountID == 0` for clarity's sake on what the goal of the conditional is.